javascript/ql/src/semmle/javascript/ApiGraphs.qll

```diff
@@ -837,9 +825,13 @@ module API {
       )
       or
       exists(string moduleName, string exportName |
+        pred = MkModuleExport(moduleName) and
+        lbl = Label::member(exportName) and
```

Comment:
With this change it seems we're again mixing up type-names and value-names which I thought we could finally avoid when we introduced `Node::ofType`.

For example, for a module with exports
```ts
export const Foo = function() { .. }
export interface Foo { ... }
```
Doing `getMember("Foo").getInstance()` will return the nodes for both `new Foo()` and nodes for expressions of type `Foo`, which seem like they should always be the same, but often are not.

To get this straight we'd either need a separate `member` edge like `typeMember` for accessing the _types_ exported from a namespace, or stop using `instance` for representing expression of a given type.